exercises/collatz-conjecture/test/Tests.hs

```diff
@@ -0,0 +1,52 @@
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
```

Comment:
I don't think this line is necessary. Can you check? If you confirm it is not necessary, please send a PR that deletes it.

Reply:
I retract. I think https://github.com/exercism/haskell/pull/144 tells me it is desirable to have for exercises with numbers.